functions/src/utils/firestoreUtils.ts

```diff
@@ -10,4 +10,16 @@ const parseDataFromSnapshot = (
   return result;
 };
 
-export { parseDataFromSnapshot };
+const getIdsFromSnapshot = (
+  snapshot: FirebaseFirestore.QuerySnapshot<FirebaseFirestore.DocumentData>
+) => {
+  const result: string[] = [];
+  snapshot.forEach((doc: FirebaseFirestore.DocumentData) => {
+    if (doc.data() !== {}) {
+      result.push(doc.id);
+    }
+  });
+  return result;
```

Comment:
why not returning directly snapshot.map?

Reply:
Because snapshot is of type `QuerySnapshot`, it's not exactly an array and it can't implement `map` method. It was my first attempt. 
That's the main reason why O did a helper fn

Reply:
snapshot.docs.map(function(doc) {}) I think this will work, not sure though